### ISSUE TITLE
Long-Press Keys do not trigger reliably in horizontal mode

### ISSUE THREAD

**Issue by SdoggaMan** [futo-org/android-keyboard#783]
### What happened?

When long pressing for numbers or symbols in the horizontal keyboard, the pop-up (which shows you the symbol and indicates that they keyboard has completed the long press and is ready to insert that alternate character) does not always insert the character, and instead inserts nothing. I believe the bug is with the long press 'selecting' the character. 

This shows you an example in Keyboard, then switching to Clash of Clans to use the horizontal keyboard: 

https://github.com/user-attachments/assets/2bfe0305-9130-433f-a1f2-3e36c673a38d

https://github.com/user-attachments/assets/6471abc6-33d4-4f42-bddb-cfc5c1a4c72d

In Vertical, I can get symbols as expected. In horizontal, you can see I only get a symbol when the 'select' highlight shows up, which happens ~40% of the time. If it doesn't 'select' the symbol, I get nothing. 

### Keyboard Version

0.1.24-playstore

**Comment by SdoggaMan** [comment]
You can see that: 
- Some long presses do not 'highlight' the symbol, and so do not insert it - but also do not insert the normal key
- Some long presses DO highlight the symbol but do not insert any character
- Some long presses work as intended

**Comment by abb128** [comment]
If you move your finger up after the symbol shows up while long-pressing, does it select the symbol consistently? If so then I might know a fix for this

**Comment by SdoggaMan** [comment]
Yep - it's like it doesn't know to ensure it selects the pop-up character when you hold it down, it needs even a pixel of input to actually register the selection. Once I discovered that I was able to mitigate the problem to a degree, but it's still pretty annoying as I tend to do my long presses fairly quickly and that room for error adds a layer of chance to hitting-or-missing the key.

**Comment by abb128** [comment]
> it needs even a pixel of input to actually register the selection

When you say that do you mean that even moving like a pixel downwards will work? Or is it only up?

**Comment by tgrelka** [comment]
I found  I can consistently replicate this, even in vertical mode, by long-pressing keys at their very bottom, below the rendered character, but still inside the touch-highlighted zone. In that case, no character in the long-press popup will be selected unless you move your finger up.

**Comment by abb128** [comment]
This will be fixed in the next release
